src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(new Random().nextInt(100));
+        ball.setColor(new ColorSupplier().getRandomColor());
```

Comment:
with this approach we create a new instance of ColorSupplier and Random per each method invocation. Let's make them class fields instead